install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       containers:
         - name: installer
           # This will normally be the release tag - using this tag as need the license evaluator
-          image: 'eu.gcr.io/gitpod-core-dev/build/installer:sje-installer-mini-config.0'
+          image: 'eu.gcr.io/gitpod-core-dev/build/installer:sje-airgapped.0'
```

Comment:
This is why I want to have a `latest` tag for the Installer

Reply:
I know. This line is actually changed by the werft job when it pushes to KOTS. It's just for avoidance for having to change line in PRs